kolena/annotation.py

```diff
@@ -382,6 +387,123 @@ class ScoredLabeledTimeSegment(TimeSegment):
     """The score associated with this time segment."""
 
 
+@dataclass(frozen=True, config=ValidatorConfig)
+class TextSegment(DataObject):
+    """
+    Segments of text in the associated text field
+    """
+
+    text_field: str
+    """Text field column name containing the text segments."""
+
+    segments: List[Tuple[int, int]]
+    """List of text segments. The first item of each tuple is the start index of the segment,
+    and the second item is the end index."""
```

Comment:
Why is this a list?

Reply:
We are going for a structure like this, top level is label, the next level is the segments for a specific field

```
[
  {
    "text_segments": [
      {
        "text_field": "text",
        "segments": [
          [
            214,
            217
          ],
          [
            724,
            727
          ]
        ]
      },
      {
        "text_field": "text_summary",
        "segments": [
          [
            72,
            75
          ],
          [
            39,
            41
          ]
        ]
      }
    ],
    "label": "pronoun",
    "color": "red",
    "data_type": "ANNOTATION/LABELED_TEXT_SEGMENTS"
  }
]
```